client/src/components/ElementEditor/FormBuilder.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import classnames from 'classnames';
+import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
+import Config from 'lib/Config';
 
-function FormBuilder() {
-  return <div className="element-editor-formbuilder">Imagine some fancy form builder here!</div>;
+function FormBuilder(props) {
+  const classNames = classnames(
+    'element-editor-formbuilder',
+    props.extraClasses
+  );
+  const baseURL = Config.get('baseUrl').replace(/\/$/, '');
+  return (<div className={classNames} onClick={props.onClick} role="presentation">
+    <FormBuilderLoader
+      formTag="div"
+      schemaUrl={`${baseURL}/admin/elemental-area/schema/elementForm/${props.elementId}`}
```

Comment:
The schema URL needs to come from configuration - you can check out the history viewer implementation for an example of this, but it's available in the Redux store as part of the global state tree.

Reason being - this could change and break here because it's not dynamic. Example: `/admin/` is a configurable path.